src/components/Tutorial/CodePreview.vue

```diff
@@ -65,11 +65,20 @@ function scaledSize({ width, height }, scale = 1) {
 
 export default {
   name: 'CodePreview',
-  inject: [
-    'references',
-    'isTargetIDE',
-    'store',
-  ],
+  inject: {
+    isTargetIDE: {
+      default: false,
+    },
+    store: {
```

Comment:
Is there a specific reason why you're not using the mixin here?

Reply:
We renamed the state `tutorialsState` in the original implementation and I wasn't sure why we did that, so I kept it the way it is.